src/rewards/rewards.v2.controller.ts

```diff
@@ -12,7 +12,6 @@ import { DebankUser } from './interfaces/debank-user.interface';
 import { ListRewardsResponse } from './interfaces/list-rewards-response.interface';
 import { RewardsService } from './rewards.service';
 
-@Deprecated()
```

Comment:
undeprecated?

Reply:
this was incorrect. this is actively used by debank